### PR TITLE
 As a user, I can access the different tabs by route #13 

### PR DESCRIPTION
**Pull request by iFrogHop2Worlds**
#### Describe your changes
I added routing with a history object for persistence and future development. I wrapped the app in a browserRouter component, and added the routes in the main page to decide which components to load depending on the route you are on. 

#### Link to issue this resolves
https://github.com/brittanyjoiner15/eui-event-template/issues/13

#### Screenshot of changes(if relevant)

* [x ] The **title** should be something descriptive about what you're changes do. _(It will default to whatever you put as your commit message.)_
* [x ] This **description** should include ...
  * [x ] A short summary of your changes
  * [x ] A link to the issue that relates to these changes
  * [ ] A screenshot of the changes if possible!
* [x ] Add `brittanyjoiner15` as a reviewer
* [x ] Add yourself as assignee

### CODE REVIEW

File: src/components/MainPage.jsx [hunk 1]
```diff
@@ -12,68 +12,45 @@ import EventDetails from "./panels/EventDetails";
 import SpeakersPanel from "./panels/SpeakersPanel";
 import TalksPanel from "./panels/TalksPanel";
 import Navbar from "./Navbar";
+import { Routes, Route } from 'react-router-dom'
+import history from "../utilities/history";
 
 function MainPage() {
-  const [selectedTab, setSelectedTab] = useState("event");
+  // const [selectedTab, setSelectedTab] = useState("event");   // commented out because approach doesnt work with routing
```
Comment (brittanyjoiner15) [inline src/components/MainPage.jsx]: You can go ahead and remove this if it's not needed anymore!

File: src/components/MainPage.jsx [hunk 2]
```diff
@@ -12,68 +12,45 @@ import EventDetails from "./panels/EventDetails";
...
-        return tabs[0].content;
-      case "speakers":
-        return tabs[1].content;
-      case "talks":
-        return tabs[2].content;
-      default:
-        return tabs[0].content;
-    }
-  };
+  // const onSelectedTabChanged = (id) => { // commented out because approache doesnt work with routing
```
Comment (brittanyjoiner15) [inline src/components/MainPage.jsx]: Same note as above - feel free to remove instead of comment out

File: src/components/MainPage.jsx [hunk 3]
```diff
@@ -12,68 +12,45 @@ import EventDetails from "./panels/EventDetails";
...
-      content: <TalksPanel />,
+      onClick: () => {history.push('/talks'); window.location.reload();},  
     },
   ];
 
-  const onLogoClick = useCallback(() => {
-    onSelectedTabChanged("event");
-  }, []);
+  
+  // const onLogoClick = useCallback(() => {  // commented out because it doesnt seem to work and conflicts with with routing
```
Comment (brittanyjoiner15) [inline src/components/MainPage.jsx]: You can remove if this wasn't needed  - it probably stopped working when you commented out the `onSelectedTabChanged` which is how we were routing content before.

File: src/components/MainPage.jsx [hunk 4]
```diff
@@ -12,68 +12,45 @@ import EventDetails from "./panels/EventDetails";
...
+  //   onSelectedTabChanged("event");         // we have a the useEffect hook running to attach an eventlistener and handle icon clicks in the App.js
+  // }, []);
 
   return (
     <EuiPage paddingSize="none">
       <EuiFlexGroup className="eui-fullHeight">
         <EuiPageBody panelled>
-          <Navbar tabs={tabs} onLogoClick={onLogoClick} />
+        <Navbar tabs={tabs} /> 
+        {/* <Navbar tabs={tabs} onLogoClick={onLogoClick} /> */}
```
Comment (brittanyjoiner15) [inline src/components/MainPage.jsx]: You can remove this

**Review by brittanyjoiner15 — CHANGES_REQUESTED** [review]
Overall looks good, but please go ahead and delete the code you commented out! I ran this locally and will test once more once you've made those changes, but should be good to go!

File: src/components/MainPage.jsx (hunk 4 above)
Comment (iFrogHop2Worlds) [reply src/components/MainPage.jsx]: ok onnit

**Comment by iFrogHop2Worlds** [comment]
comments/code were removed

**Comment by brittanyjoiner15** [comment]
@iFrogHop2Worlds looks great! fixed a merge conflict and i am merging now!

**Pull request merged**